### DISCUSSION TITLE
raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution_wsl2/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#18] [Announcements]
# raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution_wsl2/

https://uhobeike.github.io/raspicat_documentation/chat_corner/error_resolution/raspicat_error_resolution_wsl2/

**Comment by my9918** [comment]
ubuntu@ubuntu:~$ rosrun tf view_frames
Listening to /tf for 5.0 seconds
Done Listening
b'dot - graphviz version 2.43.0 (0)\n'
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/tf/view_frames", line 119, in <module>
    generate(dot_graph)
  File "/opt/ros/noetic/lib/tf/view_frames", line 89, in generate
    m = r.search(vstr)
TypeError: cannot use a string pattern on a bytes-like object

**Reply by my9918** [reply]
このサイトを参考にして解決した
https://github.com/ros/geometry/pull/193
